eos-migrate-image-defaults

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -ex
+
+old_path=/var/eos-image-defaults
+new_path=/var/lib/eos-image-defaults
+dconf_profile=/etc/dconf/profile/user
+
+if [ -d ${old_path} -a ! -h ${old_path} -a ! -e ${new_path} ]; then
+  mv ${old_path} ${new_path}
+fi
+
+if [ ! -e ${old_path} -a -d ${new_path} ]; then
+  ln -s ${new_path} ${old_path}
+fi
+
+# unfortunately the /etc/dconf/profile/user is not contained in the
+# ostree, so must be manually updated when this migration is made
```

Comment:
The image builder still creates this /etc/dconf/profile/user on every image: https://github.com/endlessm/eos-image-builder/blob/master/hooks/image/62-dconf-profile

The contents of the overrides are in the endless-image-defaults location, but this Endless-specific dconf user profile is needed to make dconf look in that path. If this Endless-specific path is included in dconf somewhere else now, then a) I didn't find it, because I searched the whole organisation for endless-image-defaults when preparing these PRs, and b) we should remove that and arrange for the file in /etc to be removed (maybe in fact make the existence of this file the trigger for this script).